docs/java.html

```diff
@@ -0,0 +1,138 @@
+                    documentation that will help you get started with your first
+                    Poker Bot Battle script. If you want the full overview of
+                    the documentation you can follow this
+                    <a
+                        href="https://poker-game-runner.readthedocs.io/en/latest/poker_game_runner.html"
+                        >link</a
+                    >. Keep in mind that the full documentation is for Python.
+                </p>
+
+                <h2>Getting Started</h2>
+                <h3>Basics</h3>
+                <p>Your program can print the following options:</p>
+                <ul>
+                    <li>
+                        0 is to fold, or check if no previous bot has raised
+                    </li>
+                    <li>
+                        1 is to call, or check if no previous bot has raised
+                    </li>
+                    <li>&gt;1 is for raising a specific amount</li>
+                </ul>
+                If your program raises an exception it will be the equivalent to
+                returning 0.
+
+                <h3>Get current cards</h3>
+                <p>
+                    This will put the bots current hand, with the table taken
+                    into account, into a variable called currentBotHand:
+                </p>
+                <pre><code class="language-java">HandType currentBotHand = obs.getMyHandType();</code></pre>
+                <p>
+                    This will put the boards hand type, with the bots hand
+                    excluded:
+                </p>
+                <pre><code class="language-java">HandType currentBoardHand = obs.getBoardHandType()</code></pre>
+                <p>
+                    Both variables will be of type HandType, which looks like
+                    this:
+                </p>
+
+                <ul>
+                    <li>STRAIGHTFLUSH = 9</li>
+                    <li>FOUROFAKIND = 8</li>
+                    <li>FULLHOUSE = 7</li>
+                    <li>FLUSH = 6</li>
+                    <li>STRAIGHT = 5</li>
+                    <li>THREEOFAKIND = 4</li>
+                    <li>TWOPAIR = 3</li>
+                    <li>PAIR = 2</li>
+                    <li>HIGHCARD = 1</li>
+                    <li>ERROR = 0</li>
+                </ul>
+                If you want to match the current handtype to a specific hand
+                type, you can do it like this:
+                <pre><code class="language-java">if (currentBotHand == HandType.STRAIGHTFLUSH) {someAction();}</code></pre>
+                <p>
+                    If you want to match the hand type to specific hands, you
+                    can do it like this:
+                </p>
+                <pre><code class="language-java">if (("77+", "A8s+", "K9s+", "QTs+", "AJo+", "KQo").isHandInRange(obs.myHand)) {someAction();}</code></pre>
```

Comment:
Det her er ikke helt rigtigt. Det er ikke en hand type du matcher til en range. Derudover er syntax lidt forkert
```suggestion
                    If you want to match you hand, to a "range" of cards, you
                    can do it like this:
                </p>
                <pre><code class="language-java">if (["77+", "A8s+", "K9s+", "QTs+", "AJo+", "KQo"]).isHandInRange(obs.myHand)) {someAction();}</code></pre>
```